dialect/sql/builder.go

```diff
@@ -701,13 +726,88 @@ func (i *InsertBuilder) Query() (string, []interface{}) {
 			i.WriteByte('(').Args(v...).WriteByte(')')
 		}
 	}
+
+	if len(i.conflictColumns) > 0 {
+		// Update on conflict
+		i.buildConflictHandling()
+	}
```

Comment:
```suggestion
	if len(i.conflictColumns) > 0 {
		i.buildConflictHandling()
	}
```

This comment is redundant here.